src/common/steps/5xFamilyVerify/verifySlice.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+import { type RootState } from '../../../app/store';
+import { setChoice } from '../../../features/device/deviceSlice';
+
+interface State {
+    response?: string;
+    error?: string;
+}
+
+const initialState: State = {};
+
+const slice = createSlice({
+    name: 'verification5x',
+    initialState,
+    reducers: {
+        setResponse: (state, action: PayloadAction<string>) => {
+            state.response =
+                state.response?.concat(action.payload) || action.payload;
+            state.error = undefined;
+        },
+        setError: (state, action: PayloadAction<string>) => {
+            state.error = action.payload;
+        },
+        onClosedPort: state => {
+            if (!state.response) {
+                state.error = 'Did not receive date from device';
```

Comment:
I guess we want to keep this passive and impersonal, like the "Failed..." notifications.
```suggestion
                state.error = 'No data received from the device';
```